clients/admin-ui/src/features/taxonomy/TaxonomyTabContent.tsx

```diff
@@ -69,7 +71,6 @@ const TaxonomyTabContent = ({ useTaxonomy }: Props) => {
     return null;
   }, [data]);
 
-  const [entityToEdit, setEntityToEdit] = useState<TaxonomyEntity | null>(null);
```

Comment:
The hooks gain a lot of info by gaining control of this state. It was necessary for looking up custom fields by fides key, but I think a later refactor could make it even ore useful.